### DISCUSSION TITLE
Death penalty

### DISCUSSION THREAD

**Discussion by avallet1** [DiscordDungeons/Bugs#990] [Ideas]
I think xp (and maybe also skill xp) loss should be added to the death penalty because to my knowledge there is no way to lose xp or skill xp as of now. 
However i think it should only start at a decided level, like level 50 or 100.

**Comment by Feathers8** [comment]
The dev considered that actually

**Comment by Sven65** [comment]
We've considered expanding the penalty of dying before to include both XP and items, but ultimately decided against it as it doesn't provide a great experience to loose your XP when dying (you're not really carrying it, are ya? :P)
